app/models/user_role.rb

```diff
@@ -58,7 +58,7 @@ def cache_user_roles!
   end
 
   def build_user_role_cache
-    [ self.cached_user_roles.build(:user => owner, :role => role) ]
+    [ self.cached_user_roles.build(:user_id => owner.id, :role_id => role.id) ]
```

Comment:
Correct, I'll remove this for the moment from here and check if it work with the 'new' validation.